### PR TITLE
Request and receive verified credentials

### PR DESCRIPTION
**Pull request by pelle**

### CODE REVIEW

File: src/Credentials.js [hunk 1]
```diff
@@ -22,6 +22,9 @@ export default class Credentials {
     if (params.requested) {
       payload.requested = params.requested
     }
+    if (params.verified) {
```
Comment (zachferland) [inline src/Credentials.js]: What is the reasoning for params.verified? and how does the response from mobile differ in these cases?

Comment (pelle) [inline src/Credentials.js]: @zachferland verified will be returned as a list of attestation JWT's from mobile

Comment (christianlundkvist) [inline src/Credentials.js]: I need to speak up about the increasing complexity here. IMO there is no reason to separate "verified" vs "unverified". I don't see any downsides to keeping it simple for now and always returning credentials as JWTs no matter who has verified them (self or other).

Comment (pelle) [inline src/Credentials.js]: @christianlundkvist the persons own credentials are not stored or created as JWT's. We need to handle them differently. Most use cases will not care about the JWT's and just want an easy way to get at their claims. The few cases that require JWT's need this.

**Pull request merged**